### PR TITLE
💂‍♀️ Admin Account Data Page

### LINKED ISSUE #211: Admin Portal Appointment Count Data
#### Description
Create page where admin can see mentor and mentees names and how many appointments they have. Admin can also download data from this page. (ignore all mentee things until we have mentee accounts up)

#### Screenshots
image

### PR DESCRIPTION
**Pull request by Leundai**
## Description :star2:

Fixes #211 
- Added table to display data
- Search functionality
- Created new endpoints and connected them
 to the backend for existing endpoints
- Styled as close to Figma
- Added animations for feedback

### Bugs:
Download Mentor Data is not working correctly, I'm not sure why I keep getting this error
image

## Screenshots:
image
 
## TODOs :star:
- [X] Tests
- [X] Documentation

### CODE REVIEW

File: frontend/src/components/pages/AdminAccountData.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+            align="center"
+          />
+          <Column
+            title="Link to Profile"
+            dataIndex="id"
+            key="id"
+            render={(id) => (
+              <a
+                style={{ color: "black" }}
+                href={formatLinkForHref(`localhost:3000/gallery/${id}`)}
```
Comment (AngelaLuo49021) [inline frontend/src/components/pages/AdminAccountData.js:161]: maybe we can have a prod env variable to toggle between the real url and localhost so its not hardcoded in?

**Review by AngelaLuo49021 — CHANGES_REQUESTED** [review]
small change request, will do full review soon tho

File: frontend/src/components/css/AdminAccountData.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
+@import "_app";
+
+.account-data-body {
+    padding: 22px;
+}
+
+.table-search {
+    width: 216px;
```
Comment (kenetec) [inline frontend/src/components/css/AdminAccountData.scss:8]: Try using `em` here

**Review by kenetec — APPROVED** [review]
gravy 👶🏿

File: frontend/src/components/pages/AdminAccountData.js (hunk 1 above)
Comment (kenetec) [reply frontend/src/components/pages/AdminAccountData.js:161]: Note: for frontend, you have to prefix env variables with `REACT_APP_`

**Review by AngelaLuo49021 — APPROVED** [review]
This isn't completely connected to backend (downloading data doesnt work) but we can circle back to that in a new issue. Good stuff leo!!!!

**Pull request merged**